lib/modules/events.js

```diff
@@ -4,7 +4,14 @@ import { findNodeHandle, NativeModules } from 'react-native';
 const { AREventsModule } = NativeModules;
 
 function postEvent(component, info) {
-  AREventsModule.postEvent(findNodeHandle(component), info);
+  var reactTag;
```

Comment:
This feels a little clumsy but without it, the AREventModule breaks because `reactTag` is undefined. Alternatively I could put `postEvent` inside the `try` block.